web-registry/src/components/organisms/CreditSendForm/CreditSendForm.schema.tsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { z } from 'zod';
+
+import {
+  invalidRegenAddress,
+  isValidAddress,
+  requiredMessage,
+} from 'web-components/lib/components/inputs/validation';
+
+export const RetireFormSchema = z.object({
+  note: z.string().optional(),
+  country: z.string(),
+  stateProvince: z.string().optional(),
+  postalCode: z.string().optional(),
+  retirementJurisdiction: z.string().optional(),
+});
+
+export type RetireFormSchemaType = z.infer<typeof RetireFormSchema>;
+
+type Params = {
+  addressPrefix?: string;
+};
+
+export const CreditSendFormSchema = ({ addressPrefix }: Params) =>
+  z.object({
+    sender: z.string(),
+    recipient: z
+      .string()
+      .refine(value => !!value, {
+        message: requiredMessage,
+      })
```

Comment:
In order to be picked up by the onBlur event we need to specify the required fields explicitly (work fine for submit by default).